modules/deploy/partials/cloud/tiers.adoc

```diff
@@ -68,6 +69,18 @@ Amazon Web Services (AWS)::
 | us-west-2
 |===
 --
+Azure::
++
+--
+|=== 
+| Region 
+
+| eastus 
```

Comment:
We have a lot of zonal failures on eastus. @c4milo and @ncole should we remove eastus from this list for now?

Reply:
^ @bpraseed

Reply:
@bpraseed please confirm that we support these Azure regions: eastus, norwayeast, uksouth

Reply:
what type of failures?

Reply:
added [centralus](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/byoc-tiers/?tab=tabs-1-azure)